src/OKRMain.tsx

```diff
@@ -25,22 +25,19 @@ const OKRPage: React.SFC<{}> = props => {
     const stateContext = useStateValue();
     // Make sure to initialize the objectives and areas
     React.useEffect(() => {
+        if (!stateContext.state.timeFrames && !stateContext.state.displayedTimeFrame) {
+            stateContext.actions.initialize(); 
+        }
```

Comment:
I want to clean up this section. Ideally, the initialize should kick off all of this, understand if its ZeroData, error, or good state, then setup the initial data appropriately.